### ISSUE TITLE
Missing `-` in TLS cipher suite docs

### ISSUE THREAD

**Issue by urso** [elastic/beats#1801]
Some cipher suite names documented do not match names in source code.

**Comment by monicasarbu** [comment]
@urso can you please give more details here?

**Comment by urso** [comment]
The configurable TLS cipher suites are available in code here: https://github.com/elastic/beats/blob/master/libbeat/outputs/tls.go#L174

It seems some names in our docs do not match the ones used by libbeat (has been pointed out by some user in the past): https://github.com/elastic/beats/blob/master/libbeat/docs/outputconfig.asciidoc#cipher_suites

names should be checked for typos.

**Comment by tsg** [comment]
Fixed by #2141. If we ever need it again, I tested the rest of a names with the following config (ciphers copied from docs):

```
output.logstash:
  # The Logstash hosts
  hosts: ["localhost:5044"]

  tls.cipher_suites: ["ECDHE-ECDSA-AES-256-GCM-SHA384", "ECDHE-RSA-AES-256-GCM-SHA384", "ECDHE-ECDSA-AES-128-GCM-SHA256",
    "ECDHE-RSA-AES-128-GCM-SHA256", "ECDHE-RSA-AES-256-CBC-SHA", "ECDHE-RSA-AES-128-CBC-SHA", "ECDHE-RSA-3DES-CBC3-SHA",
    "ECDHE-RSA-RC4-128-SHA", "ECDHE-ECDSA-AES-256-CBC-SHA", "ECDHE-ECDSA-AES-128-CBC-SHA", "ECDHE-ECDSA-RC4-128-SHA",
    "RSA-AES-256-CBC-SHA", "RSA-AES-128-CBC-SHA", "RSA-3DES-CBC3-SHA", "RSA-RC4-128-SHA"]
```
